cmd/src/setup.rs

```diff
@@ -90,24 +83,83 @@ pub fn initial_logger(config: &TiKvConfig) {
             )
             .unwrap_or_else(|e| {
                 fatal!(
-                    "failed to initialize log with file {}: {}",
+                    "failed to initialize slow-log with file {}: {}",
                     config.slow_log_file,
                     e
                 );
             });
-            let drainer = logger::LogDispatcher::new(drainer, slow_log_drainer);
-            logger::init_log(
-                drainer,
-                config.log_level,
-                true,
-                true,
-                vec![],
-                config.slow_log_threshold.as_millis(),
-            )
-            .unwrap_or_else(|e| {
-                fatal!("failed to initialize log: {}", e);
-            });
+            Some(slow_log_drainer)
         };
+
+        let rocksdb_info_log_dir = if config.rocksdb.info_log_dir.is_empty() {
+            config::canonicalize_sub_path(&config.storage.data_dir, DEFAULT_ROCKSDB_SUB_DIR)
```

Comment:
CI is failing because `fs::canonicalize` require path exists. Let's call `fs::create_dir_all` before canonicalize.

Reply:
@hawkingrei